packages/bpk-component-chip/examples.js

```diff
@@ -200,6 +268,10 @@ const DismissibleChipsExample = () => (
 
 const SelectableChipsExample = () => <StatefulSelectableChips />;
 
+const RadiogroupChipsExample = () => (
+  <StatefulRadioGroupChipsExample type={CHIP_TYPES.primar} />
```

Comment:
```suggestion
  <StatefulRadioGroupChipsExample type={CHIP_TYPES.primary} />
```

Reply:
Thanks :) I think parts of my editor experience is broken since this should've been caught. Haven't done web in a long time